azure-stack/hci/manage/manage-syslog-forwarding.md

```diff
@@ -0,0 +1,469 @@
+---
+title: Manage syslog forwarding for Azure Stack HCI (preview)
+description: Learn about security features for Azure Stack HCI security information and event management (SIEM) (preview).
+author: alkohli
+ms.author: alkohli
+ms.topic: how-to
+ms.service: azure-stack
+ms.subservice: azure-stack-hci
+ms.date: 12/01/2023
+---
+
+# Manage syslog forwarding for Azure Stack HCI (preview)
+
+[!INCLUDE [hci-applies-to-23h2](../../includes/hci-applies-to-23h2.md)]
+
+This article describes how to manage components in the security layer of the Azure Stack HCI, version 23H2 solution for security information and event management (SIEM).
+
+For additional security considerations, see:
+
+- [Security baseline settings on Azure Stack HCI (preview)](/azure-stack/hci/concepts/secure-baseline).
+- [BitLocker encryption on Azure Stack HCI (preview)](/azure-stack/hci/concepts/security-bitlocker).
+
+[!INCLUDE [important](../../includes/hci-preview.md)]
+
+## About local built-in user accounts
+
+In this release, the names of local built-in users associated with the `RID 500` and `RID 501` accounts are updated. The following table provides details of the built-in accounts:
+
+|Name |Enabled |Description |
+|-----|-----|-----|
+|ASBuiltInAdmin |True |Built-in account for administering the computer/domain |
+|ASBuiltInGuest |False |Built-in account for guest access to the computer/domain |
+
+> [!Important]
+> We recommend that you create your own local administrator account, and also that you disable the well-known `RID 500` user account.
+
+The `ASBuiltInGuest` account is protected by the security baseline drift control mechanism. For more information, see [Security baseline settings for Azure Stack HCI (preview)](/azure-stack/hci/concepts/secure-baseline).
+
+We also added two net new components to Azure Stack HCI in 23H2 release, as follows:
+
+- Secrets management engine.
+- SIEM syslog forwarder agent.
+
```

Comment:
Remove the local builtin user accounts section from here. It will be in the conceptual article.

Reply:
Done > removed.